client/sites-dashboard-v2/hooks/use-sync-selected-site-feature.ts

```diff
@@ -44,6 +44,11 @@ export function useSyncSelectedSiteFeature( {
 
 		if ( page.current !== newUrl ) {
 			page.show( newUrl );
+
+			const featurePreviewContentElement = document.querySelector( '.item-preview__content' );
```

Comment:
I think we can make changes to `ItemPreviewPane` directly as A4A might need this feature as well 🙂

Reply:
> However, this might also be fragile since it depends on a class name that could be changed. Any suggestions?

We can use `ref` in `ItemPreviewPane` as follows. As @arthur791004 mentioned above, it will benefit A4A as well.

```diff
@@ -1,4 +1,4 @@
-import { ReactNode } from 'react';
+import { ReactNode, useRef, useEffect } from 'react';
 
 import './style.scss';
 
@@ -8,5 +8,19 @@ type Props = {
 };
 
 export default function ItemPreviewPaneContent( { children }: Props ) {
-	return <div className="item-preview__content">{ children }</div>;
+	const ref = useRef();
+
+	useEffect( () => {
+		if ( ref.current ) {
+			window.setTimeout( () => {
+				ref.current.scrollTop = 0;
+			} );
+		}
+	} );
+
+	return <div className="item-preview__content" ref={ ref }>{ children }</div>;
 }
```

Reply:
Fair! Let me work on this some more 🙂

Reply:
> I think we can make changes to ItemPreviewPane directly as A4A might need this feature as well 🙂 cc @cleacos

Yes!!! We had that issue/task to do, too. So perfect! Thank you.